media-libs/libarib25/libarib25-20180717.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="Implementation of ARIB STD-B25"
+HOMEPAGE="https://github.com/stz2012/libarib25"
+
+MY_PV="22fcf5f6062317a603bf407344492483a24512ed"
```

Comment:
It is the worst way of taking VCS snapshots and should never be used due to possible further official releases, because it is going to confuse portage, You should take a tarball snapshot of the master tree (or whatever else you want and upload it somewhere), or at least rename yout ebuild to `libarib25-0_pre20180717.ebuild` and handle this properly.

Reply:
Upstream released v0.2.5-20180719, so ebuild rename to libarib25-0.2.5.20180719.ebuild.
Is it OK?
https://github.com/stz2012/libarib25/releases